composer/_version.py

```diff
@@ -3,4 +3,4 @@
 
 """The Composer Version."""
 
-__version__ = '0.24.0.dev0'
+__version__ = '0.23.3'
```

Comment:
isn't this supposed to still be 0.24.dev0, and then only on the release branch you change it to 0.23.0? pretty sure that's what's in the doc

Reply:
@mvpatel2000 updated the doc to fix the release note auto-generation. Now it needs this change and an extra PR to restore the version to dev